install/kubernetes/helm/istio/charts/sidecar-injector/templates/clusterrole.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1beta1
 kind: ClusterRole
 metadata:
   name: istio-sidecar-injector-{{ .Release.Namespace }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Again, pending confirmation this doesn't break Tiller.

Reply:
I tried helm install and it is not failing Tiller.

Reply:
confirmed